tests/dragonfly/replication_test.py

```diff
@@ -2695,6 +2695,48 @@ async def test_replication_timeout_on_full_sync_heartbeat_expiry(
     await assert_replica_reconnections(replica, 0)
 
 
+@pytest.mark.exclude_epoll
+@dfly_args({"proactor_threads": 1})
+async def test_big_string(df_factory):
```

Comment:
We could just `DEBUG POPULATE` big string, save to rdb, shutdown df and reload. Why bother with a replication test here ? It's *very* similar to `test_big_value_serialization_memory_limit`

Reply:
static seeder used in this test is debug populate , but I can change this to use debug populate directly
I don't think that using replication is a bother, we want to make sure that we don't have a spike in memory when loading the data. From my perspective using replica or if we do save and than load is simillar.